benchmark/benchmark-atomic.cpp

```diff
@@ -0,0 +1,220 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+// RAJA/RAJA.hpp cannot be included here because the include logic will
```

Comment:
Comment is inconsistent with the code below. RAJA/RAJA.hpp is included.

Perhaps it would be best to include a comment block describing what the code in the file does, how to run it, and how it is organized. For example, running both desul and non-desul atomics in a single executable is not something that a typical RAJA user would do.